docs/workflow.rst

```diff
@@ -77,10 +76,11 @@ Git Branching Off
 
 Usually, a new branch should be branched off from a target to-be-merged remote branch.
 It is often *upstream/develop* or sometimes *upstream/master*. However, there are cases
-which are not applied:
-Always keep in mind that you need to rebase often the work of that upstream branch to your working branch.
+which are not applied.
+
+Remember to rebase your workspace branch with the upstream branch as often as possible.
 
-Firstly, you must know what the meaning of “Branching Off” is . In a shorthand, it means you checkout from a branch,
+Firstly, you must know what the meaning of “Branching Off” is. In a shorthand, it means you checkout from a branch,
```

Comment:
In case, you are working on a branch which depends on many different branches => improve this sentence.